metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/extraction/AudioVideoProcessor.java

```diff
@@ -160,6 +165,25 @@ public ResourceExtractionResult process(Resource resource) throws MediaExtractio
     return new ResourceExtractionResult(metadata, null);
   }
 
+  private static int findInt(String key, JSONObject[] candidates) {
+    final int defaultValue = Integer.MIN_VALUE;
+    return Stream.of(candidates).map(candidate -> candidate.optInt(key, defaultValue))
```

Comment:
Just use the Integer.MIN_VALUE in both usages of defaultValue instead.

Reply:
Fixed.